### DISCUSSION TITLE
can new posts be bundled? (in a new subfolder)

### DISCUSSION THREAD

**Discussion by drwilkins** [estruyf/vscode-front-matter#140] [Q&A] [answered]
I'm using [doks](https://getdoks.org/) to create my site's documentation and blog. It expects blog entries to be bundled in a new subfolder; e.g. blog/new-blog-entry/, which contains index.md and any images.

Can this structure be created using vscode-front-matter when I click "Create Content" on the Dashboard?

**Comment by estruyf** [comment, marked as answer]
Hi @drwilkins, not at this moment, but it would be a good enhancement to Front Matter to enable it. Could be a configuration/setting that specifies if you want to create content as a single file, or bundle.

**Reply by estruyf** [reply]
BTW: is this for all types of content you want to create?

**Reply by drwilkins** [reply]
Thanks, that would be awesome!

Using this particular Hugo theme, the blog would need to be bundled, but the docs seem to have multiple .md posts within the same subfolder.

i.e.:
```
|- blog
    |- post_1
        |- index.md
        |- post_pic.png
    |- post_2
        |- index.md
        |- post_pic.png
|- docs
    |- help (list page)
        |- faq.md
        |- how-to-update.md
        |- _index.md
    |- quickstart (list page)
        |- commands.md
        |- introduction.md
        |- _index.md
    |- privacy-policy (single page)
        |- index.md  
```
